src/document.h

```diff
@@ -413,12 +413,10 @@ struct Document {
     auto CopyEntireCells(wxString &s) {
         sys->clipboardcopy = s;
         wxString html = selected.g->ConvertToText(selected, 0, A_EXPHTMLT, this, false);
-        auto htmlobj = 
-        #ifdef __WXGTK__
-            new wxCustomDataObject(wxDF_HTML);
-            htmlobj->SetData(html.Len(), html);
+        #if wxCHECK_VERSION(3, 3, 0)
+            auto htmlobj = new wxHTMLDataObject(html);
         #else
-            new wxHTMLDataObject(html);
+            static_assert(false, "wxWidgets < 3.3.0 is not supported.");

```

Comment:
why is this in this function? this can be at a global level. With the check inside the assert.

Reply:
Ok, I changed that. Is it ok now?

Reply:
Yup, perfect :)